docs/DeveloperGuide.md

```diff
@@ -250,24 +250,77 @@ The activity diagram shows the general sequence of steps when a user interacts w
 
 #### Implementation
 
+1. **Input Parsing**  
+   The user command input is parsed in `LogicManager` by calling `AddressBookParser#parseCommand`. For a command starting with "find person", the parser creates a `FindPersonCommandParser` instance to parse specific parameters.
```

Comment:
Should we remove this input parsing part for each feature, since it would be covered under the general implementation of the command at the start of the section? (you can refer to https://github.com/AY2425S1-CS2103T-W10-2/tp/pull/186 for the general implementation of the entity commands, currently being worked on)